OneSignalExample/Assets/OneSignal/README.md

```diff
@@ -1,35 +1,112 @@
-## OneSignal Push Notifications
-[OneSignal](https://www.onesignal.com) is a free push notification service for Android and iOS apps. This SDK makes it easy to integrate your Unity app with OneSignal.
+<p align="center">
+  <img src="https://media.onesignal.com/cms/Website%20Layout/logo-red.svg"/>
+</p>
 
-### Supported Platforms
-#### Devices
-* iOS 9 - 14
-* Android 4.0.3 (API Level 15) - Android 11 (API Level 30)
-* Amazon FireOS 3 - 6
+# OneSignal Unity SDK
+[OneSignal](https://onesignal.com) is a free push notification service for mobile apps. This SDK makes it easy to integrate your Unity app with OneSignal.
```

Comment:
It's more than for mobile. I think it's worth stating all that we do for devs who get introduced to the product through this package.

Reply:
This is just a copy/paste of the copy that was already present. However I definitely agree that might be worth noting separately as this description is directly regarding the functionality contained within this package. Maybe something like

> A free push notification and in app messaging solution for mobile applications built through Unity. [OneSignal](https://onesignal.com) provides a fully array of omni-channel messaging solutions across:
> - Push Notifications (mobile & web)
> - In App Messages
> - SMS
> - Email
> 
> and via many additional platforms. [Check them all out](https://documentation.onesignal.com/docs/sdk-reference)!

Reply:
👍 Suggestion looks good to me!

Reply:
Fixed in c623914.